webapp/web/selector.css

```diff
@@ -0,0 +1,19 @@
+#project-selector select {
+    width: 150px;
+    margin: 4px 10px;
+    display: flex;
+    box-sizing: border-box;
+    padding: 0.5em 2em 0.5em 0.5em;
+    border: 1px solid #eee;
+    font: inherit;
+    line-height: inherit;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    -ms-appearance: none;
+    appearance: none;
+    background-color: #f3f3f3;
+    background-repeat: no-repeat;
+    background-image: linear-gradient(45deg, transparent 50%, currentColor 50%), linear-gradient(135deg, currentColor 50%, transparent 50%);
+    background-position: right 15px top 1em, right 10px top 1em;
+    background-size: 5px 5px, 5px 5px;
+  }
```

Comment:
nit: needs empty last line, and the closing bracket seems to be indented to 2 spaces rather than 0?

Reply:
Fixed that